helm-chart/todoapp/charts/mysql/templates/configMap.yml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Chart.Name }}-config
+  namespace: {{ .Values.common.namespace }}
```

Comment:
Ensure that the `namespace` value in `{{ .Values.common.namespace }}` is correctly defined in your `values.yaml` file. If `common.namespace` is not set, this could lead to deployment issues.